lib/ui/fake_flutter/collections.dart

```diff
@@ -16,13 +16,10 @@ library collections;
 ///
 ///   * [listEquals], which does something similar for lists.
```

Comment:
I wouldn't format any of the code in fake_flutter but this is fine. generally try to keep wholesale format operation runs in separate CLs as that makes the change history easier to review and understand.

Reply:
reverted.